### PR TITLE
(GH-iac-334) Remove code specific to unsupported OSs

### PR DESCRIPTION
**Pull request by david22swan**
Unsure of whether this should be merged now as the last release backwards incompatible and this will make two in a row.
In the event that it is we should ensure that as many of the community prs are brought into it as possible.

### CODE REVIEW

**Comment by ekohl** [comment]
> Unsure of whether this should be merged now as the last release backwards incompatible and this will make two in a row.

I think we should at least look at https://github.com/puppetlabs/puppetlabs-apache/pull/2202. Either we should move forward or (my preference) we should remove the warnings.

**Comment by david22swan** [comment]
I would not be opposed to removing the warning, highly doubt the plan is going to be moved forward at this point as everyone involve with it is now gone to the best of my knowledge

**Review by ekohl — COMMENTED** [review]
Looks like I'm too late (had a review, forgot to submit), but you introduced a regression.

File: manifests/mpm.pp [hunk 1]
```diff
@@ -98,17 +98,13 @@
       }
 
       if $mpm == 'prefork' {
-        if ( ( $::operatingsystem == 'Ubuntu' and versioncmp($::operatingsystemrelease,'18.04') >= 0 ) or $::operatingsystem == 'Debian' ) {
```
[the remark this answers is not in the document]
Comment (chelnak) [reply manifests/mpm.pp:101]: Hey! Good spot. We will investigate asap!

[the remark this answers is not in the document]
Comment (david22swan) [reply manifests/mpm.pp:101]: @ekohl The change should be fine as it is within the Debian section of the `case $::osfamily` started on line 50.
So it shouldn't effect any non Debian os'es

[the remark this answers is not in the document]
Comment (ekohl) [reply manifests/mpm.pp:101]: Ah good point. I'm not used to such long if/else. I just wondering if it should be restructured.

**Pull request merged**